client/src/plugins/update-checks/__tests__/UpdateChecksSpec.js

```diff
@@ -295,6 +295,36 @@ describe('<UpdateChecks>', function() {
 
   it('should check periodically (every N minutes)');
 
+
+  it('should check with URL encoded parameters', async () => {
+
+    // given
+
+    Flags.init({
+      [ UPDATE_SERVER_URL ]: 'http://test-update-server.com'
+    });
+
+    const {
+      component
+    } = createComponent();
+
+    let calledURL = '';
+
+    mockServerResponse(component, {
+      update: {
+        latestVersion: 'v3.7.0',
+        downloadURL: 'test-download-url',
+        releases: []
+      }
+    }, (url) => { calledURL = url; });
+
+    // when
+    await tick(component);
+
+    // then
+    expect(calledURL).to.eql('http://test-update-server.com/update-check?editorID=test-id&newerThan=v3.5.0&modelerVersion=v3.5.0&os=windows&osVersion=98&productName=Camunda+Modeler&plugins%5Bid%5D=plugin1&plugins%5Bname%5D=plugin1&plugins%5Bid%5D=plugin2&plugins%5Bname%5D=plugin2');
```

Comment:
Is there any difference between `Camunda+Modeler` and `Camunda%20Modeler` as URI param?

Reply:
I googled this, they are kinda the same, implementation specific. Both URL safe. We use the URL API, so we're good there.

Reply:
I'm asking because of this:
![image](https://user-images.githubusercontent.com/28307541/86365599-01c5e580-bc7a-11ea-87c3-63014910dfc5.png)

Reply:
On the other hand, `URLSearchParams` use `+` here.

Let's leave it then :)

Reply:
I'll double check to make sure.

Reply:
👍

Reply:
I made this request:
`"/update-check?editorID=813332dc-73c9-4e56-a05e-1e345ddd86d4&newerThan=v4.0.0&modelerVersion=v4.0.0&os=darwin&osVersion=18.7.0&productName=Camunda+Modeler"` and checked the /events data. We only have "Camunda Modeler" but no "Camunda+Modeler", so we're good to go 👍

Reply:
That's great.